source/extensions/common/aws/signer.h

```diff
@@ -23,9 +23,11 @@ class Signer {
   /**
    * Sign an AWS request.
    * @param headers AWS API request headers.
+   * @param use_unsigned_payload use the literal string `UNSIGNED-PAYLOAD` to calculate the payload
+   * hash.
    * @throws EnvoyException if the request cannot be signed.
    */
-  virtual void sign(Http::RequestHeaderMap& headers) PURE;
+  virtual void sign(Http::RequestHeaderMap& headers, bool use_unsigned_payload = false) PURE;
```

Comment:
No default argument for virtual functions:
https://google.github.io/styleguide/cppguide.html#Default_Arguments

Reply:
@lizan Can you please offer a concrete suggestion? I don't want to misinterpret this rather terse comment and spend unnecessary effort making changes that don't directly address your concern.